model/bindings.go

```
@@ -47,81 +43,87 @@ func CreateBindingsRancher(pathToYml string) BindingProperty {
 		yamlContent, err := ioutil.ReadFile(dockerFile)
 		if err != nil {
 			log.Errorf("Error in opening file : %v\n", err)
-		} else {
-			err = yaml.Unmarshal(yamlContent, &rawConfigDocker)
-			if err != nil {
-				log.Errorf("Error during Unmarshal for file %s : %v\n", dockerFile, err)
-			} else {
-				rawConfigDocker, err := preprocess.PreprocessServiceMap(rawConfigDocker)
-				utils.Convert(rawConfigDocker, &configMap)
-				if err != nil {
-					log.Errorf("Error")
-				}
-			}
+			return nil, err
+		}
+		err = yaml.Unmarshal(yamlContent, &rawConfigDocker)
+		if err != nil {
+			log.Errorf("Error during Unmarshal for file %s : %v\n", dockerFile, err)
+			return nil, err
+		}
+		rawConfigDocker, err = preprocess.PreprocessServiceMap(rawConfigDocker)
+		if err != nil {
+			log.Errorf("Error")
```

Can we return the error here?